### DISCUSSION TITLE
Open word document in the same folder

### DISCUSSION THREAD

**Discussion by soutienti** [SafeExamBrowser/seb-win-refactoring#801] [Q&A] [answered]
Hello
I want to use the SEB for open a word document. 

![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/159486156/25699b3c-409a-4ddc-a4ff-e0a7b8bf20f2)

the word document is in the same folder than the seb file

![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/159486156/a9751085-10d7-4f31-ae72-b6696153f951)

but when I open the it say sorry we can't find 
c:\examen.docx

when I entered the exact path it work, but it will be different for each student. if we can't use the sometime like .\ or \ 
is possible ? 

thanks a lot for your help

**Comment by dbuechel** [comment, marked as answer]
If you were to use e.g. the Downloads directory of each user, you might be able to get something working with the usage of an enviornment variable: `%USERPROFILE%\Downloads`.

**Comment by soutienti** [comment]
thanks for your answer, it's a little bit complicate but your school is bring your own laptop, the laptop wasn't connected to the AD (you have no idea how this makes my job more complicated) 
the complete path is \\server\level\username in the AD\
so the username it's not the same as in the laptop

Is there a way to have a relative path to the file?
thanks for you time

**Reply by dbuechel** [reply]
I would reckon there still could be a way, but that is an organizational resp. technical issue with respect to your institution to which I cannot really comment unfortunately.

**Reply by soutienti** [reply]
do you have an idea ? 
I try .\
\
examen.docx 
but they can't find the file so they use the c:\windows

**Reply by dbuechel** [reply]
Terribly sorry, apart from the aforementioned there is not much I could contribute here from afar.

**Reply by soutienti** [reply]
thanks for your time, I find a another way. 
thanks

**Reply by dbuechel** [reply]
Excellent! Please do feel free to share it here in case you'd think that other users could benefit from your findings.

**Reply by soutienti** [reply]
I used windows variables, as you suggested. but a little differently, I'm start the seb by a batch file
set sebexam=%~dp0
%sebexam%examen\examen.seb

in the SEB application option I use my temporary variable. 
it's not the better way, but it's working.
